pkg/aws/client.go

```diff
@@ -613,13 +614,54 @@ func (c *awsClient) FilterVPCsPrivateSubnets(subnets []ec2types.Subnet) ([]ec2ty
 		}
 	}
 
+	// Temporary filter until local-zone and wave-length zones are supported by HCP
+	if isHostedCp {
+		privateSubnets, err = c.FilterSubnetsWithStandardAvailabilityZones(privateSubnets)
+		if err != nil {
+			return nil, err
+		}
+	}
+
 	if len(privateSubnets) < 1 {
 		return nil, fmt.Errorf("failed to find private subnets associated with VPC '%s'", *subnets[0].VpcId)
 	}
 
 	return privateSubnets, nil
 }
 
+func (c *awsClient) FilterSubnetsWithStandardAvailabilityZones(subnets []ec2types.Subnet) ([]ec2types.Subnet, error) {
```

Comment:
Does this func need to be exported?

Reply:
Nope the function does not need to be exported.
Was following unit test conventions in the client_test file where the methods being tested were called through the "Client" interface, instead of a mock client.

I will go ahead and change this to a non exportable method.